jenkins/migrationIntegPipelines/solutionsCFNTestCover.groovy

```diff
@@ -0,0 +1,9 @@
+def gitBranch = params.GIT_BRANCH ?: 'sol-pipeline'
+def gitUrl = params.GIT_REPO_URL ?: 'https://github.com/lewijacn/opensearch-migrations.git'
```

Comment:
Does this need to be updated?

Reply:
Yes should be swapped back before merging, easier for testing right now